templates/common/infra/bicep/core/host/appservice.bicep

```diff
@@ -33,19 +33,21 @@ param numberOfWorkers int = -1
 param scmDoBuildDuringDeployment bool = false
 param use32BitWorkerProcess bool = false
 param ftpsState string = 'FtpsOnly'
+param httpLoggingEnabled bool = false
 param healthCheckPath string = ''
 
 resource appService 'Microsoft.Web/sites@2022-03-01' = {
   name: name
   location: location
-  tags: tags
+  tags: union(tags, { 'azd-service-name': 'web' })
```

Comment:
This doesn't seem right - these core modules should not contain `azd` specific tags.  This module might be used by multiple services and so the `azd-service-name` tag should be passed into this module (as a member of it's `tags` property)

Reply:
matt is correct